### ISSUE TITLE
How to use flasgger with Blueprint

### ISSUE THREAD

**Issue by vladimircape** [rochacbruno/flasgger#159]
Hello.
I have mainApp.py and right now use Vagrant
```
from flask import Flask
import logging
from controllers.MainController import main_api
import settings,os
from flasgger import Swagger


#connect controllers
application = Flask(__name__)
Swagger(application)
application.register_blueprint(main_api)
```

and in MainController.py
```
 -*- coding: utf-8 -*-
from flask import Blueprint, jsonify, current_app,send_file
import sys
from time import gmtime, strftime
from flasgger.utils import swag_from
from flasgger import Swagger

main_api = Blueprint('main', __name__)
main_api.config = {}
#Swagger(main_api)

@main_api.route("/startranking", methods=['GET'])
@swag_from("documentation/startranking.yml")
def startRanking():
```
But got error 

> File "/usr/local/lib/python3.6/site-packages/flasgger/utils.py", line 456, in load_from_file site_package = imp.find_module(path[0])[1] File "/usr/local/lib/python3.6/imp.py", line 296, in find_module raise ImportError(_ERR_MSG.format(name), name=name) ImportError: No module named 'vagrant' --> http://192.168.50.27:5000/apispec_1.json

**Comment by rochacbruno** [comment]
@vladimircape Flasgger works ok with Blueprints as in the test located in https://github.com/rochacbruno/flasgger/blob/master/examples/example_blueprint.py 

Your error has nothing to do with flasgger but with your Python installation

`ImportError: No module named 'vagrant'`

**Comment by vladimircape** [comment]
@rochacbruno Thanks for fast reply.
I saw this example, but i have little bit different. my MainController.py it's a separate python file.
I made like in you sample just in one file mainApp.py and everything was ok, but when i start to separate files got this error.
About error. It looks very strange, because Vagrant it's my https://www.vagrantup.com/ which i use for this application.
This error i got when i open xxx.xxx.xxx.xxx:5000/apidoc .
